packages/components/src/templates/next/components/internal/BlogCard/BlogCard.tsx

```diff
@@ -84,9 +86,11 @@ export const BlogCard = ({
           </Text>
         )}
         {/* TODO: Feature enhancement? Filter by category when clicked */}
-        <Text className="prose-label-sm-medium text-base-content-subtle">
-          {category}
-        </Text>
+        {shouldShowCategory && (
```

Comment:
issue: I think we should still show the category even if it is the only one? When clicking the link to open the article, the category will still be shown on the article page, so initial thoughts is that not showing the category here makes the category on the article page a little out of place.

Reply:
we only not show category when there's no category filter on the lefthand side (since that's where `shouldShowCategory` is derived from), which means there's either no/one category. IMO will clutter the UI on the collection results since it then serves no purpose in this scenario

re: comment from Sehyun https://opengovproducts.slack.com/archives/C07CWUNUL68/p1735803802912019?thread_ts=1735799448.795359&cid=C07CWUNUL68